src/views/__tests__/Onboarding.test.tsx

```diff
@@ -217,16 +220,36 @@ const executeStep1 = async (partyName: string) => {
   await waitFor(() => expect(fetchWithLogsSpy).toBeCalledTimes(3));
 };
 
+const executeStepInstitutionType = async () => {
+  console.log('Testing step Institution Type');
+  await waitFor(() => screen.getByText(stepInstitutionType));
+
+  await checkBackForwardNavigation(step1Title, stepInstitutionType);
+  await fillInstitutionTypeCheckbox(
+    'pa' as OrganizationType,

```

Comment:
These values should be upperCase.
If the type is using lowercase, let's fix it in order to use uppercase :)